exercises/strings/replaceCharacter/replaceCharacter.js

```diff
@@ -12,15 +12,28 @@
  * @returns {string} A copy of the input string with target replaced by replaceWith
  */
 
-function replaceCharacter(string, target, replaceWith) {
-  // This is your job. :)
+function replaceCharacter(string, replace, replaceWith) {
+  let stringArray = [];
```

Comment:
You can do:

```javascript
let stringArray = string.split('');
```

Reply:
Would recommend using a name that better reflects the contents of the variable, e.g.,

```javascript
let characters = string.split('');
```